src/pybind/mgr/zabbix/module.py

```diff
@@ -231,9 +240,17 @@ def send(self):
                                   self.config['zabbix_port'], self.log)
 
             zabbix.send(identifier, data)
+            self.set_health_checks(dict())
             return True
         except Exception as exc:
             self.log.error('Exception when sending: %s', exc)
+            self.set_health_checks({
+                'MGR_ZABBIX_SEND_FAILED': {
+                    'severity': 'warning',
+                    'summary': 'Failed to send data to Zabbix due to a Exception',
```

Comment:
It would leave off the "due to a Exception here" -- the message conveys all the useful information without it.